<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QMainWindow" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/myappico.png</normaloff>:/myappico.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAjouter_un_evenement"/>
    <addaction name="actionVoir_les_evenements_enregistres"/>
    <addaction name="actionSupprimer_un_evenement"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjouter_un_evenement">
   <property name="text">
    <string>Ajouter un evenement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionVoir_les_evenements_enregistres">
   <property name="text">
    <string>Voir les evenements enregistres</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSupprimer_un_evenement">
   <property name="text">
    <string>Supprimer un evenement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
